src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,49 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
+        if (!file_exists($composerLockPath)) {
+            $installedPackages = array();
+        } else {
+            $composerLock = json_decode(file_get_contents($composerLockPath), true);
+            foreach (array_merge($composerLock['packages'], $composerLock['packages-dev']) as $package) {
+                $installedPackages[$package['name']] = $package['version'];
+            }
+        }
+
+        foreach ($this->kernel->getBundles() as $name => $bundle) {
+            $bundlePath = $bundle->getPath();
+
+            $bundleVersion = null;
+            foreach ($installedPackages as $packageName => $packageVersion) {
+                if (preg_match(sprintf('~.*/vendor/%s~', preg_quote($packageName)), $bundlePath)) {
```

Comment:
This doesn't work if the bundle class isn't at the root of the package (or if the bundle use the target-dir composer feature). You could use ```'~.*/vendor/%s.*~'```, this would remove the need to treat Symfony packages separately below.